### PR TITLE
update jotai to v1.6.0  and correct example to make it work fine

### PR DESCRIPTION
**Pull request by tsingson**
update example to make its work correct.

### CODE REVIEW

File: examples/hacker_news/src/App.tsx [hunk 1]
```diff
@@ -2,29 +2,33 @@ import { Suspense } from 'react'
 import { a, useSpring } from '@react-spring/web'
 import Parser from 'html-react-parser'
 import { Provider, atom, useAtom, useSetAtom } from 'jotai'
+import { atomWithQuery } from 'jotai/query'
```
Comment (dai-shi) [inline examples/hacker_news/src/App.tsx:5]: We don't want to change this example with RQ. If you need such a example, please create a new one.

**Review by dai-shi — CHANGES_REQUESTED** [review]
Thanks for working on this. A quick first review.

**Comment by dai-shi** [comment]
> to make its work correct.

what was the incorrect example, previously?

**Comment by TwistedMinda** [comment]
@tsingson Hey tsingson, do you think you will be able to complete this? 

I can take care of updating to last version & fixing the problem, but do you remember what was not working in the first place?

The [Example CodeSandBox](https://codesandbox.io/s/github/pmndrs/jotai/tree/main/examples/hacker_news?file=/src/App.tsx) seems to be working just fine

**Comment by TwistedMinda** [comment]
@dai-shi I guess we can close it and we'll rework examples if there is a bug found or we make a major update. Correct me if I'm wrong.
Most important is that the example is working fine.

OP can still comment here if he has an issue

**Pull request closed without merging**